common/src/main/java/org/opensearch/ml/common/transport/memorycontainer/memory/MLGetMemoryAction.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.common.transport.memorycontainer.memory;
+
+import org.opensearch.action.ActionType;
+
+public class MLGetMemoryAction extends ActionType<MLGetMemoryResponse> {
+    public static final MLGetMemoryAction INSTANCE = new MLGetMemoryAction();
+    public static final String NAME = "\"cluster:admin/opensearch/ml/memory_containers/memory/get";
```

Comment:
is this `\"` needed? you can remove it. 

check out the MLTaskGetAction

https://github.com/opensearch-project/ml-commons/blob/de3df61394b7477047aa7d33eb34fdb107440cea/common/src/main/java/org/opensearch/ml/common/transport/task/MLTaskGetAction.java#L12